pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -84,6 +85,17 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to cloud providers.
+	err = c.Watch(
```

Comment:
Looks like watches are being used to add regularly occurring events onto the queue. Cool use-case.